app/views/discovery_rules/_form.html.erb

```diff
@@ -6,7 +6,27 @@
                      :control_group_id => 'search_group', :size => 'col-md-9' %>
   <%= select_f f, :hostgroup_id, accessible_hostgroups, :id, :to_label, { :include_blank => false },
     { :help_inline => _('Target host group for this rule with all properties set')} %>
-  <%= text_f f, :hostname, :help_inline => _('Target hostname template pattern (empty for MAC)')  %>
+  <%= text_f f, :hostname, :help_inline => popover(
+    _("Template"),
```

Comment:
Well actually if I do that then the booble will be narrow because the more text I add to the link the less space remains. If you have a solution how to change the arrow direction I can do that.